source/_integrations/solax.markdown

```diff
@@ -17,6 +17,12 @@ The `solax` integration connects Home Assistant to Solax solar power inverters.
 
 ## Configuration
 
+To configure the connection with your inverter simply configure it via the UI with the Solax integration. You will be asked for the IP Address, the port and password (optional).
```

Comment:
Remove the whole configuration section and replace it with :

```markdown
{% include integrations/config_flow.md %}
```

Please also remove the YAML configuration from the documentation, as that should have been deprecated when moving an integration to the UI.